src/ocean/time/StopWatch.d

```diff
@@ -141,6 +144,24 @@ unittest
 
     // prove that stopwatch time at least counts up ...
     nanosleep(&wait_time, null);
+    auto stopwatch_sec = t.sec;
+    test!(">")(stopwatch_sec, 0);
+
+    // prove that calling `sec` doesn't stop anything ...
```

Comment:
Fair enough. Yeah maybe just change the comment to `Validate that checking the elapsed time doesn't stop the timer`?